phpBB/includes/acp/acp_styles.php

```diff
@@ -259,6 +259,19 @@ protected function action_uninstall()
 		// Get list of styles to uninstall
 		$ids = $this->request_vars('id', 0, true);
 
+		// Don't remove prosilver, you can still deactivate it.
+		$sql = 'SELECT style_id
+			FROM ' . STYLES_TABLE . "
+			WHERE style_name = '" . $this->db->sql_escape('prosilver') . "'";
+		$result = $this->db->sql_query($sql);
+		$prosilver_id = (int) $this->db->sql_fetchrow($result);
```

Comment:
Does it actually return the  ```style_id```? I think ```sql_fetchrow``` returns an array. But ```sql_fetchfield``` would return a value directly 😉

Reply:
Indeed, same here.. that's old code like

```
		// Check if style has child styles
		$sql = 'SELECT style_id
			FROM ' . STYLES_TABLE . '
			WHERE style_parent_id = ' . (int) $id . " OR style_parent_tree = '" . $this->db->sql_escape($path) . "'";
		$result = $this->db->sql_query($sql);

		$conflict = $this->db->sql_fetchrow($result);
		$this->db->sql_freeresult($result);
```

I will use your suggestion though, just is a bit late here.